src/docs/components/preview-style-select.svelte

```diff
@@ -21,29 +21,38 @@
 		typeof v === 'string' && (value = v);
 	});
 	$: localValue.set(value);
+
+	beforeNavigate(() => {
+		localValue.set('tailwind');
+	});
 </script>
 
 <button
-	class="flex w-[140px] items-center justify-between rounded-md border border-magnum-700 bg-neutral-800 px-3 py-1 text-magnum-600
-  outline-none transition-opacity hover:opacity-75 focus:!border-magnum-400 focus:!text-magnum-400"
+	class="flex w-[140px] items-center justify-between rounded-md border border-magnum-700 bg-neutral-800 px-3 py-1
+  text-magnum-600 outline-none transition-opacity hover:opacity-75 focus:!border-magnum-400 focus:!text-magnum-400"
 	{...$trigger}
 	use:trigger
 	aria-label="Select"
 >
-	{$label || 'Select an option'}
+	{#if $label === 'css'}
+		CSS
+	{:else if $label === 'tailwind'}
+		Tailwind
+	{:else}
+		Select an option
+	{/if}
```

Comment:
So the object it's pulling these labels from sets its props based on the filenames, in this case `tailwind.svelte` and `css.svelte`, I suppose I could take it a step further since we'll only have `tailwind` or `css` and just handle that in the util function that builds the object to get rid of this mess!

I only did it this way because CSS is all caps and tailwind is first letter caps 😆